Listener/Notifier.php

```diff
@@ -137,23 +138,40 @@ public function onKernelException(GetResponseForExceptionEvent $event)
             }
         }
     }
+    
+    /**
+     * Handle the console exceptions (Symfony 2.3-3.4)
+     *
+     * @param ConsoleExceptionEvent $event event
+     */
+    public function onConsoleException(ConsoleExceptionEvent $event)
+    {
+        $this->handleConsoleException($event->getException());
+    }
 
     /**
-     * Handle the event
+     * Handle the console errors (Symfony 4+)
      *
      * @param ConsoleErrorEvent $event event
      */
     public function onConsoleError(ConsoleErrorEvent $event)
     {
-        $exception = $event->getError();
-
+        $this->handleConsoleException($event->getError());
+    }
+    
+    /**
+     * Common handling logic for console exceptions
+     * @param \Throwable $exception
+     */
+    private function handleConsoleException(\Throwable $exception)
+    {
         $sendMail = !in_array(get_class($exception), $this->ignoredClasses);
 
         if ($sendMail === true) {
             $this->createMailAndSend($exception, null, null, $this->command, $this->commandInput);
         }
     }
-
+    
```

Comment:
Should be reverted as well :)

Reply:
Not sure what you want reverted here, as this class needs to be modified.

Reply:
I meant reverting the changes on this line and l.141, i.e introducing 4 spaces instead of a blank line.